torchtune/utils/_distributed.py

```diff
@@ -8,11 +8,17 @@
 import logging
 import os
 from itertools import chain
-from typing import Callable, Dict, Set, Tuple, Type
+from typing import Any, Callable, Dict, Set, Tuple, Type
 
 import torch
 import torch.distributed as dist
 from torch import nn
+
+try:
```

Comment:
Sorry for the inconvenience on this rename 😢

Reply:
it's all good :D

Reply:
Since we depend on new code changes anyway for DCP, should we just enforce at least a certain pytorch version and avoid this `try`/`except`?

Reply:
I removed the depdency on newer changes in DCP since there is still gaps on sd["state"], numerics, "initial_lr". FSDP is now the only one. I am open to pytorch version though if it feels more statisfied than `try/except`

Reply:
that’s a good one. will try to find a version from pytorch nightly